src/klein/test/test_exports.py

```diff
@@ -0,0 +1,69 @@
+"""
+Tests for exported API.
+"""
+
+from twisted.trial import unittest
+
+
+
+class PublicSymbolsTestCase(unittest.TestCase):
+    """
+    Tests for public API modules.
+    """
+
+    def test_klein(self):
+        # type: () -> None
+        """
+        Test exports from L{klein}.
+        """
+        import klein as k
+        import klein._app as a
+        import klein._plating as p
+
+        self.assertIdentical(k.Klein, a.Klein)
+        self.assertIdentical(k.handle_errors, a.handle_errors)
+        # Blocked on https://github.com/twisted/klein/issues/224
+        # self.assertIdentical(k.resource, a.resource)
+        self.assertIdentical(k.route, a.route)
+        self.assertIdentical(k.run, a.run)
+
+        self.assertIdentical(k.Plating, p.Plating)
+
+
+    def test_app(self):
+        # type: () -> None
+        """
+        Test exports from L{klein.app}.
+        """
+        import klein.app as a
+        import klein._app as _a
+
+        self.assertIdentical(a.Klein, _a.Klein)
+        self.assertIdentical(a.KleinRequest, _a.KleinRequest)
+        self.assertIdentical(a.handle_errors, _a.handle_errors)
+        self.assertIdentical(a.route, _a.route)
+        self.assertIdentical(a.run, _a.run)
+
+
+    def test_interfaces(self):
+        # type: () -> None
+        """
+        Test exports from L{klein.interfaces}.
+        """
+        import klein.interfaces as i
+        import klein._interfaces as _i
+
+        self.assertIdentical(i.IKleinRequest, _i.IKleinRequest)
+
+
+    # Blocked on https://github.com/twisted/klein/issues/224
+    # def test_resource(self):
```

Comment:
Same issue with commented out code.  Let's use GitHub's inter-PR/issue linking to record this metadata.